tools/toolchain/prepare

```diff
@@ -32,28 +34,61 @@ if reg digest $(<tools/toolchain/image) > /dev/null; then
     exit 1
 fi
 
-archs=(amd64 arm64)
+if !(type git-archive-all > /dev/null 2>&1); then
+    if [ "$ID" = "fedora" ]; then
+        sudo dnf install -y git-archive-all
+    else
+        echo install git-archive-all
+        exit 1
+    fi
+fi
 
 # docker arch has a diffrent spelling than uname arch
-declare -A arch_unames=(
-    [amd64]=x86_64
-    [arm64]=aarch64
+declare -A docker_arch=(
+    [x86_64]=amd64
+    [aarch64]=arm64
     [s390x]=s390x
 )
 
-for arch in "${archs[@]}"; do
-    # translate from docker arch to uname arch
-    arch_uname="${arch_unames[$arch]}"
-    if [[ "$(uname -m)" == "${arch_uname}" ]]; then
-        continue
-    fi
-    if [[ ! -f  /proc/sys/fs/binfmt_misc/qemu-"${arch_uname}" ]]; then
-        echo install qemu-user-static
-        exit 1
-    fi
+usage() {
+    cat <<EOF
+
+Options:
+  --clang-install          build and install an optimized clang compiler binary.
```

Comment:
install into dbuild, not the host I reckon?

Reply:
tools/toolchain/prepare is for preparing the frozen toolchain.

Reply:
But we do want the ability to install to the host (can just call clang_lto_pgo.sh directly)